packages/cheatsheet/src/lib/sampleSpokenFormInfos/defaults.json

```
@@ -158,6 +158,16 @@
             }
           ]
         },
+        {
+          "id": "followLinkAside",
+          "type": "action",
+          "variations": [
+            {
+              "spokenForm": "follow aside <target>",
+              "description": "Follow link in a split"
+            }
+          ]
+        },
```

This is supposed to be generated automatically, but it's not easy to trigger today. For now I'd just manually change it to what the generated output would be
```suggestion
        {
          "id": "followLinkAside",
          "type": "action",
          "variations": [
            {
              "spokenForm": "follow split <target>",
              "description": "Follow link aside"
            }
          ]
        },
```